packages/ui-link/src/Link/index.tsx

```diff
@@ -79,12 +79,23 @@ class Link extends Component<LinkProps, LinkState> {
   }
   ref: Element | null = null
 
+  checkProps = () => {
+    const { variant, isWithinText } = this.props
+
+    warn(
+      !!variant && !!isWithinText,
```

Comment:
This does not work. First this needs to be `false` for a warning to display and second it will not work since we default `isWithinText` to `true`, so it displays the warning by default. I think it can be removed